### PR TITLE
Fix #509: Make iProov frame parsing null-safe

### LINKED ISSUE #509: Make iProov frame parsing null-safe

### PR DESCRIPTION
**Pull request by banterCZ**

### CODE REVIEW

File: enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/presencecheck/iproov/provider/IProovPresenceCheckProvider.java [hunk 1]
```diff
@@ -251,8 +248,17 @@ private <T> T parseResponse(String body, Class<T> cls) throws PresenceCheckExcep
         }
     }
 
-    private String unescapeSlashes(String value) {
-        return value.replace("\\", "");
+    private static Image parseImage(final ClaimValidateResponse response) {
+        final String frameJpeg = unescapeSlashes(response.getFrame());
```
Comment (romanstrobl) [inline enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/presencecheck/iproov/provider/IProovPresenceCheckProvider.java:252]: I suggest to add a null check for `response`.

**Review by romanstrobl — CHANGES_REQUESTED** [review]
I suggest to add one paranoid null check, the error is really hard to debug.

File: enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/presencecheck/iproov/provider/IProovPresenceCheckProvider.java (hunk 1 above)
Comment (banterCZ) [reply enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/presencecheck/iproov/provider/IProovPresenceCheckProvider.java:252]: Sure

**Review by romanstrobl — APPROVED** [review]
Great.

**Pull request merged**